packages/about-you/composables/__tests__/composables/useLocale/index.spec.ts

```diff
@@ -0,0 +1,47 @@
+import useLocale from '../../../src/composables/useLocale';
+
+jest.mock('@vue/composition-api', () => ({
+  getCurrentInstance: () => ({
+    $i18n: {
+      locales: [
+        { locale: 'de', country: 'de', currency: 'eur'},
+        { locale: 'en', country: 'us', currency: 'usd'},
+        { locale: 'pl', country: 'pl', currency: 'pln'}
+      ],
+      setLocaleCookie: jest.fn(() => true),
+      setLocale: jest.fn(() => true)
+    }
+  }),
+  ref: jest.fn()
+}));
+
+describe('[about-you-cloud composables] useLocale', () => {
+  describe('availableCountries', () => {
+    it('returns array with available locales', () => {
+      const { availableCountries } = useLocale();
+      expect(availableCountries).toEqual(['de', 'us', 'pl']);
+    });
+  });
+
+  describe('availableCurrencies', () => {
+    it('returns array with available currencies', () => {
+      const { availableCurrencies } = useLocale();
+      expect(availableCurrencies).toEqual(['eur', 'usd', 'pln']);
+    });
+  });
+
+  describe('setLocale', () => {
+    it('returns mocked setLocaleI18n value', () => {
+      const { setLocale } = useLocale();
+      expect(setLocale('en')).toBeTruthy();
+    });
+  });
+
+  describe('setCookie', () => {
+    it('returns mocked setLocaleCookie value', () => {
+      const { setCookie } = useLocale();
+      expect(setCookie('cookieValue')).toBeTruthy();
```

Comment:
toBeTrue would fit better here. We should use toBeTruthy/Falsy rather rarely. Anyway, don't you think that some real values (strings) instead of true would be better?